cmd/operator-sdk/run/local.go

```diff
@@ -70,10 +70,12 @@ func (c *runLocalArgs) addToFlags(fs *pflag.FlagSet) {
 }
 
 func (c runLocalArgs) run() error {
-	log.Infof("Running the operator locally in namespace %s.", c.watchNamespace)
-
-	projutil.MustInProjectRoot()
-
+	// The new layout will not have c.watchNamespace
+	if kbutil.HasProjectFile() {
+		log.Infof("Running the operator locally ...")
+	} else {
+		log.Infof("Running the operator locally in namespace %v.", c.watchNamespace)
```

Comment:
Using the word "in" is confusing. The operator can't be both running locally and running in a namespace.

I think you mean that the operator is _watching_ the namespace.
```suggestion
		log.Infof("Running the operator locally; watching namespace %q", c.watchNamespace)
```